### ISSUE TITLE
Support of wallabag v2

### ISSUE THREAD

**Issue by jlnostr** [SSilence/selfoss#713]
Yes, I know there's support for wallabag. But it's just for the v1 version, not for the v2 version.
In the future it could be possible to click on the wallabag menu entry and the link is sent directly to wallabag. No need for login, no external window.

Would be really awesome, if [selfoss could support wallabag v2](https://www.wallabag.org/blog/2015/10/06/developers-oauth2) :)

_And yes, I know that wallabag v2 is still in progress._

**Comment by thatscoteng** [comment]
I made the pull request that put wallabag v1 support in. I did this by using the bookmark provided by wallabag to save pages. 

Not sure my skills extend much further than that.
I have not had a look at wallabag v2 yet. I also now have a small child which I didn't have when first adding support so have even less time now. 

TL;DR: Not started to look and not sure I am capable.

**Comment by jtojnar** [comment]
This was included in 2.17 using `wallabag_version` option. The one-click saving is best implemented using [plug-in system](#877), once we have it.
